tobac/feature_detection.py

```diff
@@ -981,6 +1341,9 @@ def filter_min_distance(
     y_coordinate_name=None,
     z_coordinate_name=None,
     target="maximum",
+    PBC_flag="none",
+    max_h1=0,
```

Comment:
I believe that this is now fixed. It would probably be better to use `*args **kwargs` for passing to this function...

Reply:
Please double check my work here though!

Reply:
Hi @freemansw1!  If I am not mistaken, there are still 4 missing parameters: `x_coordinate_name`, `y_coordinate_name`, `min_h1` and `min_h2`. Do you think these are parameters should be optionally defined by the user? If so, we would need to make them also input parameters for `feature_detection_multithreshold` wherein `filter_min_distance` is called since this is the function used by users. 

If these input parameters were only set for potential internal use, I would say it makes sense they still need to be updated here even when they are set to the default values or what is the general approach to update function calls of enhanced functions? 

Would `*args **kwargs` make a big difference here given that we still need to define `args`? 

All this will be less of a struggle when when we move to base classes for the main functions I guess :)

Reply:
Also: `min_h1` and `min_h2` do not appear in the doc strings

Reply:
All right, thanks for your further explanations @freemansw1! That makes sense and I am happy with that solution for now